x-pack/plugins/enterprise_search/public/applications/enterprise_search_overview/components/product_selector/product_selector.tsx

```diff
@@ -54,32 +45,53 @@ export const ProductSelector: React.FC<ProductSelectorProps> = ({
 }) => {
   const { hasAppSearchAccess, hasWorkplaceSearchAccess } = access;
   const { config } = useValues(KibanaLogic);
+  const { errorConnectingMessage } = useValues(HttpLogic);
+
+  const showErrorConnecting = !!(config.host && errorConnectingMessage);
+  // The create index flow does not work without ent-search, when content is updated
+  // to no longer rely on ent-search we can always show the Add Content component
+  const showAddContent = !showErrorConnecting && config.host;
```

Comment:
If my memory of formal logic doesn't fail me, this is the equivalent of `(config.host && !errorConnectingMessage)`, which I think is a bit easier to understand (there's an ent-search and it's not throwing errors).

Reply:
Sounds good, hopefully we can de-couple Indices page soonish and remove this clause all together.